libs/ledger-live-common/src/device-core/customLockScreen/screenSpecs.ts

```diff
@@ -0,0 +1,87 @@
+import { DeviceModelId } from "@ledgerhq/devices";
+import {
+  CLSSupportedDeviceModelId,
+  supportedDeviceModelIds,
+} from "../capabilities/isCustomLockScreenSupported";
+
+type ScreenSpecs = {
+  /* width of the screen in pixels */
```

Comment:
Not sure if we have a convention about the comments in LL, but this [doc](https://google.github.io/styleguide/tsguide.html#jsdoc-vs-comments) could give some inspirations.

Reply:
ah great catch it was meant to be JSDoc, thanks!